modules/ROOT/pages/upgrade-message-flows.adoc

```diff
@@ -10,15 +10,26 @@ Runtime templates related to Partner Manager are upgraded when:
 * Functional and performance defects are resolved in the template
 * Flow optimizations are implemented
 
-When new runtime templates are available, the *Upgrade deployments* button next to the search bar in the *Message Flow* page is enabled.
+When new runtime templates are available, the *Manage deployments* button is enabled with these options:
```

Comment:
The 'Manage deployments' button is always enabled. It is the 'Upgrade deployments' radio button that gets enabled when new runtime templates are available.